### DISCUSSION TITLE
Does it make a difference to Directus which database is used?

### DISCUSSION THREAD

**Discussion by ghost** [directus/directus#17016] [Archive]
Is there a difference for directus between using `MySQL 8` or `MySQL 5` or `PostgreSQL` (or any other supported database). Are there any features that can only be used with Postgres – or MySQL? Is one better than the other for directus?

Is there any difference?

**Comment by rijkvanzanten** [comment]
Nope[^1]! It's personal preference / business requirements. The app/API functionality is identical between the vendors. Which one you want to use is solely based on which database you prefer using yourself, which one your host has available, or which one you already have to use due to other business requirements. 

[^1]: For PostgreSQL, make sure to install PostGIS to get the geolocation functionality as well.
